### DISCUSSION TITLE
How to use this with tailwind plugins

### DISCUSSION THREAD

**Discussion by beaussan** [tw-in-js/twind#16] [Q&A]
Hi,

First of all, your project looks great! I was wondering if there is a way to use tailwind plugins like typography or forms with this library, or we have to implement it ourself?

Thanks

**Comment by sastan** [comment]
We have started to port typography as [@twind/typography](https://github.com/tw-in-js/typography). You can find an example in https://github.com/tw-in-js/example-svelte. There are some features missing (like theming of prose).

Forms is still open but planned.

For compatibility reasons we are thinking about if we could support tailwindcss plugins within twind. We can not support the whole plugin API but the parts (80%) that are most commonly used maybe possible.

**Comment by sastan** [comment]
I have created an issue #32 to track what needs to be done to support tailwind plugins. Feel free to add your input.

**Reply by beaussan** [reply]
It looks great !
